src/angularComponent/ngcOmniboxController.js

```diff
@@ -184,11 +184,11 @@ export default class NgcOmniboxController {
   }
 
   /**
-   * Select a particular suggestion item.
+   * Adds a particular suggestion item to the list of selected items.
    *
    * @param {Object} item
    */
-  selectItem(item) {
+  addToSelections(item) {
```

Comment:
I think `addToSelections` is a good mirror to `removeFromSelections`. I don't think it matters when multiselect is off, it does the same thing, just removes the one selection of one. From what I've seen when a function can perform an action on one or more, the usually just pluralize it.